frontend/src/metabase/query_builder/components/notebook/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -95,20 +97,30 @@ export function NotebookDataPicker({
 
   return (
     <>
-      <UnstyledButton
-        c="inherit"
-        fz="inherit"
-        fw="inherit"
-        p={NotebookCell.CONTAINER_PADDING}
-        disabled={isDisabled}
-        onClick={handleClick}
-        onAuxClick={handleAuxClick}
+      <Tooltip
+        label={t`${METAKEY}+click to open in new tab`}
+        hidden={!table}
+        events={{
+          hover: true,
+          focus: false,
+          touch: false,
+        }}
```

Comment:
@mngr reported that the tooltip was present when you come back to the notebook. This prop fixes that scenario.

To check:
1. CMD click to open in new tab
2. Tooltip disappears
3. Go to the new tab
4. Close the new tab
5. You should now be in the notebook window/tab again
6. Tooltip should not be open

Reply:
And here's a test for it: https://github.com/metabase/metabase/pull/46552/commits/f1949d68ea80189c474bbeae930cfcb4cb07d7e0